guidelines.md

```diff
@@ -77,4 +125,24 @@ You can utilize our existing SDKs functions to handle access token
 
 Stuck somewhere in implementation, Raise a support ticket. We'll be happy to help you.
 
-![Image](support_ticket.png)
\ No newline at end of file
+![Image](support_ticket.png)
+
+
+
+## FAQs
+
+1. How we are reading the users Access Token?
+
+After successful login from LoginRadius IDX, the access token is returned as the query parameter to the callback URL, Access token is been read from the callback URL and stored in the browser.
+
+
+2. How logout is working if we do logout on IDX page?
+
+After Logout action is performed on the IDX successfuly, user is redirected to the callback URL. On the callback page, we are checking if the access token validity. If invalid, Access Token is been removed from the localStorage and Cookies. 
```

Comment:
change to `After Logout action is performed on the IDX successfully, the user is redirected to the callback URL. On the callback page, we are checking if the access token is valid. If invalid, Access Token is removed from the localStorage and Cookies. `